underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -405,6 +405,9 @@ private String stripFolderSuffixIfPresent(String key) {
    * @return the key without the Swift container prefix
    */
   private String stripPrefixIfPresent(String path) {
+    if (path.startsWith(PATH_SEPARATOR)) {
+      return stripPrefixIfPresent(path, PATH_SEPARATOR);
+    }
```

Comment:
Does the Swift API not support querying any paths which start with a leading `PATH_SEPARATOR`?